build.sbt

```diff
@@ -51,14 +51,14 @@ parallelExecution in Test := false
 val AkkaVersion = "2.4.1"
 
 libraryDependencies ++= Seq(
-  "com.datastax.cassandra"  % "cassandra-driver-core"               % "3.0.0-alpha5",
+  "com.datastax.cassandra"  % "cassandra-driver-core"               % "3.0.0-rc1",
   "com.typesafe.akka"      %% "akka-persistence"                    % AkkaVersion,
   "com.typesafe.akka"      %% "akka-persistence-query-experimental" % AkkaVersion,
   "com.typesafe.akka"      %% "akka-persistence-tck"                % AkkaVersion      % "test",
   "org.scalatest"          %% "scalatest"                           % "2.1.4"      % "test",
   "com.typesafe.akka"      %% "akka-stream-testkit-experimental"    % "1.0"        % "test",
   // cassandra-all for testkit.CassandraLauncher, app should define it as test dependency if needed
-  "org.apache.cassandra"    % "cassandra-all"                       % "3.0.0"      % "optional"
+  "org.apache.cassandra"    % "cassandra-all"                       % "3.0.1"      % "optional"
```

Comment:
Actually, I cannot find any newer driver published yet (at the time of writing).
http://mvnrepository.com/artifact/com.datastax.cassandra/cassandra-driver-core